client/editor/tool_tile.cpp

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: GPLv3-or-later
+// SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+
+// Qt
+#include <QPainter>
+
+// Editor
+#include "editor/tool_tile.h"
+
+// Client
+#include "fcintl.h" //editor_tool_tile
+#include "fonts.h"  //editor_tool_tile
+
+/**
+ *  \class editor_tool_tile
+ *  \brief A widget that allows a user to edit tiles on maps and scenarios.
+ *
+ *  As of March 2025, this is a brand new widget.
+ */
+
+/**
+ *  \brief Constructor for editor_tool_tile
+ */
+editor_tool_tile::editor_tool_tile(QWidget *parent)
+{
+  auto value_font = fcFont::instance()->getFont(fonts::chatline);
+  ui.setupUi(this);
+  parent = 0;
```

Comment:
This ain't doing anything (sets the `parent` variable to a null pointer, but you never use it again)
```suggestion
```